PennyPincher/PennyPincherAndroidGesturesImporter.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  PennyPincherAndroidGesturesImporter.swift
+//  PennyPincher
+//
+//  Created by Raf Cabezas on 1/30/17.
+//  Copyright © 2017 WillowTree, Inc. All rights reserved.
+//
+
+import Foundation
+
+// Example usage:
+//
+//    for gesture in PennyPincherAndroidGesturesImporter.translatedGestures(from: PennyPincherAndroidGesturesImporter.defaultAndroidFilePath ?? "") {
+//        if let template = PennyPincher.createTemplate(gesture.id, points: gesture.allPoints) {
+//            pennyPincherGestureRecognizer.templates.append(template)
+//        }
+//    }
+//
+
+public struct ImportedStrokes {
+    public let points: [CGPoint]
+    
+    public init(points: [CGPoint]) {
+        self.points = points
+    }
+}
+
+public struct ImportedGesture {
+    public let id: String
+    public let strokes: [ImportedStrokes]
+    
+    public init(id: String, strokes: [ImportedStrokes]) {
+        self.id = id
+        self.strokes = strokes
+    }
+    
+    public var allPoints: [CGPoint] {
+        return strokes.reduce([CGPoint]()) { points, stroke in
+            return points + stroke.points
+        }
```

Comment:
Funny you say that, I actually did exactly that (flatMap) before I added allPoints, then copied allPoints from the example GestureView/ points struct (code above). Thanks.